### PR TITLE
Update rubygem-foreman_remote_execution to 2.0.2

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.24

Replacement PR for https://github.com/theforeman/foreman-packaging/pull/4207

---

### CODE REVIEW

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -9,42 +9,70 @@
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
+%{!?_root_libexecdir:%global _root_datadir %{_libexecdir}}
```
Comment (zjhuntin) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: This doesn't seem to be used. Is it needed?

**Comment by zjhuntin** [comment]
This PR will break: tfm-rubygem-foreman_ansible & tfm-rubygem-foreman_salt
Can we get PRs that will fix these so we can merge them all at the same time?

**Comment by lzap** [comment]
Requested dependencies updates, this will require GEM releases as well.

**Comment by lzap** [comment]
https://github.com/theforeman/foreman-packaging/pull/4250

**Comment by sbernhard** [comment]
Use #4256, please

**Pull request merged**